src/components/wire/wire.js

```diff
@@ -0,0 +1,45 @@
+import { div } from '@cycle/dom'
+import xs from 'xstream'
+import delay from 'xstream/extra/delay'
+
+/**
+ *
+ * @param STREAM$ any Stream
+ * @param props$ : startPosition{x, y}, radius, length, color, animateColor
+ * @returns {{STREAM$: *}}
+ */
+export default ({ STREAM$, props$ }) => {
+  const className = '.wire'
```

Comment:
You are listening to only one stream.
But if we listen to all stream, we could determine the colors (standard and animate) from the stream.

NOTE$ => black / red
MUSIC$ => black / green
etc